### ISSUE TITLE
What to do with "Overview" section of hazard

### ISSUE THREAD

**Issue by shadkeene** [weather-gov/weather.gov#712]
### Description

Not sure if this is a bug or enhancement, but some products include an "overview" section, like the section circled below.  The current beta.weather.gov does not include the "overview" section.  I'm not sure if this was intended or not, or if the "overview" section is not available on the API.

### Acceptance criteria

We are all aware of the "overview" section and we know how to deal with that when it is available.

- [ ] Tests have been implemented or modified to capture the bug
- [ ] Changes have been tested for accessibility
![overview](https://github.com/weather-gov/weather.gov/assets/139174561/ebb7c6b3-9544-4e96-95bd-08e84f830d66)
![no_overview](https://github.com/weather-gov/weather.gov/assets/139174561/eb90faa5-a2e9-4b43-ba20-f72c008a83c5)

**Comment by greg-does-weather** [comment]
Here's what we're getting from the API. It looks like the overview is in here. We'll probably need to dig into the formatting a little more to ensure this is consistent.

Also noticing the area description in this alert isn't in the format we expect. We expect locations separated by semicolons. On this one, it's `Lassen-Eastern Plumas-Eastern Sierra Counties`.

```
"description": "...WINTER CONDITIONS RETURN TO THE SIERRA AND NORTHEAST CALIFORNIA
FOR MID-LATE WEEK...

.After a few days of warm weather, a potent winter storm will bring
windy and colder conditions with periods of heavy snow to the Sierra
and higher elevations of northeast California later this week. While
weather-related travel impacts aren't expected through Wednesday
morning, conditions will begin to worsen Wednesday afternoon and
evening, with the most widespread winter travel impacts likely from
Wednesday evening through much of Thursday.

* WHAT...Heavy snow possible. Snow accumulations of 4 to 10 inches
above 5000 feet west of US-395, with 10 to 20 inches possible for
higher passes such as Fredonyer Summit and Yuba Pass. Winds
gusting as high as 50 mph.

* WHERE...Lassen-Eastern Plumas-Eastern Sierra Counties.

* WHEN...From late Wednesday morning through Friday morning.
Heaviest snow is most likely from late Wednesday afternoon through
Thursday morning.

* IMPACTS...Travel could be very difficult at times, with hazardous
conditions impacting the commutes from Wednesday evening through
Friday morning. Strong winds may blow down some tree limbs and a
few power outages may result.

* ADDITIONAL DETAILS...Snow levels will start near 6500 feet on
Wednesday, then drop to near 5500 feet Wednesday night and near
5000 feet by Thursday morning."
```

**Comment by greg-does-weather** [comment]
The full API response for that alert:

[active__status=actual&point=40.3237,-121.0172.json](https://github.com/weather-gov/weather.gov/files/14090347/active__status.actual.point.40.3237.-121.0172.json)

**Comment by colinmurphy01** [comment]
Tagging @kmranjo for awareness

**Comment by colinmurphy01** [comment]
@shadkeene can you identify which products include an `overview` section (if not all of them)?

**Comment by kmranjo** [comment]
Quick question here to make sure I'm understanding: Overview is not a separate field, but additional text within the "description" field, correct? Reading through the output for this example, it looks like it is included within "description". 
If that's the case, then yes this is a bug because the expectation is that everything in the description field should display. If it conforms to using only the what/where/when/etc headers then great, but it shouldn't get cut off if it doesn't use those.

**Comment by colinmurphy01** [comment]
@greg-does-weather created #716 to track the location issue separately. We need to determine if we should be supporting this method of separating locations on our end or if this is a one-off.

**Comment by greg-does-weather** [comment]
Related to #713

**Comment by greg-does-weather** [comment]
Closed by #748
